src/patternfly/components/Button/examples/button-states-example.hbs

```diff
@@ -68,14 +68,14 @@
 {{/button}}
 <br><br>
 {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Remove"'}}
-  {{> button-icon button-icon--type="times"}}
+  {{> button-icon button-icon--type="times" button--attribute='aria-label="Remove"'}}
 {{/button}}
 {{#> button button--modifier="pf-m-plain pf-m-focus" button--attribute='aria-label="Remove"'}}
-  {{> button-icon button-icon--type="times"}}
+  {{> button-icon button-icon--type="times" button--attribute='aria-label="Remove"'}}
 {{/button}}
 {{#> button button--modifier="pf-m-plain pf-m-active" button--attribute='aria-label="Remove"'}}
-  {{> button-icon button-icon--type="times"}}
+  {{> button-icon button-icon--type="times" button--attribute='aria-label="Remove"'}}
 {{/button}}
 {{#> button button--modifier="pf-m-plain" button--attribute="disabled" button--attribute='aria-label="Remove"'}}
```

Comment:
All of the changes made so far in this file should be removed. 

I _think_ the issue is here on this line, we have `button--attribute` defined twice, and it looks like it's ignoring the second one where `aria-label` is defined. I'm not sure how to concatenate attributes here. Maybe the following would work?:
`button--attribute='disabled aria-label="Remove"'`
I that doesn't work, then maybe @mcoker can provide a suggestion.

Reply:
Spoke with @mcoker about the consolidating the `disabled` `button--attribute` together with the `aria-label` and this is the result. Not really sure if the `disabled` actually works.

<img width="889" alt="Screen Shot 2019-03-13 at 11 37 08 AM" src="https://user-images.githubusercontent.com/1402829/54292602-dd299400-4584-11e9-8f4e-ff8556ad7ed8.png">